src/Components/Cards/Cards.css

```diff
@@ -184,18 +184,107 @@
   font-size: 1.3rem;
 }
 
+.banner-card {
+  padding: 0;
+  max-width: 58rem;
+  height: auto;
+  width: 100%;
+  border-radius: 20px;
+  border: none;
+  margin: auto;
+}
+
+.banner-card img {
+  width: 100%;
+  margin: 0;
+  border-radius: 20px;
+}
+
+.banner-card h1 {
+  position: absolute;
+  color: var(--text-white);
+  background-color: var(--primary-color);
+  width: auto;
+  padding: 1rem;
+  border-radius: 0 0 20px 20px;
+  font-size: 1.3rem;
+  text-align: center;
+  font-weight: 500;
+  left: 50%;
+  transform: translate(-50%, 0);
+  bottom: 0;
+  width: 100%;
+}
+
+.category-card {
+  border-radius: 20px;
+  width: 9rem;
+  height: 7.8rem;
+  padding: 0;
+  margin: 0 10px;
+  margin-bottom: 10px;
+  background-color: var(--card-hover-bg-color);
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
+
+.category-card:hover {
+  background-color: var(--card-overlay);
+  color: var(--text-white);
+}
+
+.category-text-section {
+  text-align: center;
+}
+
+.category-card i,
+.category-card h1 {
+  font-size: 1.1rem;
+  color: var(--title-color);
+  margin: 6px 0;
+}
+
 .icon-inactive {
   color: var(--icon-color);
 }
 
-@media screen and (max-width: 580px) {
+@media screen and (max-width: 1024px) {
+  .category-card {
+    border-radius: 20px;
```

Comment:
use rems here

Reply:
For border-radius, it should be in pixel and not in rem or em. Because the radius should be consistent throughout any size for root font.